config/database.yml

```diff
@@ -23,3 +23,7 @@ test:
 production:
   <<: *default
   database: db/production.sqlite3
+
+staging:
+  <<: *default
+  database: db/staging.sqlite3
```

Comment:
Just checked now. I think Heroku uses it's on config for production database. I think we still need to fix our database.yml for example https://github.com/andela-iamadi/zhishi-backend/blob/master/config/database.yml#L24 still uses the default configuration for production and our default is sqlite